deploy/openshift/releases/R2008/manifests/0000000-contrail-09-manager.yaml

```diff
@@ -0,0 +1,263 @@
+              image: hub.juniper.net/contrail-nightly/contrail-statusmonitor:2008.121
+          ipFabricForwarding: false
+          ipFabricSnat: true
+          kubernetesTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
+          useKubeadmConfig: true
+          hostNetworkService: true
+    vrouters:
+    - metadata:
+        labels:
+          contrail_cluster: cluster1
+        name: vroutermasternodes
+      spec:
+        commonConfiguration:
+          create: true
+          imagePullSecrets:
+            - contrail-registry
+          nodeSelector:
+            node-role.kubernetes.io/master: ""
+        serviceConfiguration:
+          cassandraInstance: cassandra1
+          controlInstance: control1
+          contrailStatusImage: hub.juniper.net/contrail-nightly/contrail-status:2008.121-rhel
+          containers:
+            - name: init
+              image: python:3.8.2-alpine
+            - name: nodeinit
+              image: hub.juniper.net/contrail-nightly/contrail-node-init:2008.121-rhel
+            - name: vrouteragent
+              image: hub.juniper.net/contrail-nightly/contrail-vrouter-agent:2008.121-rhel
+            - name: vroutercni
+              image: hub.juniper.net/contrail-nightly/contrail-kubernetes-cni-init:2008.121-rhel
+            - name: vrouterkernelbuildinit
+              image: hub.juniper.net/contrail-nightly/contrail-vrouter-kernel-build-init:2008.121-rhel
+            - name: vrouterkernelinit
+              image: hub.juniper.net/contrail-nightly/contrail-vrouter-kernel-init:2008.121-rhel
+            - name: multusconfig
+              image: busybox:1.31
+    - metadata:
+        labels:
+          contrail_cluster: cluster1
+        name: vrouterworkernodes
+      spec:
+        commonConfiguration:
+          create: true
+          imagePullSecrets:
+            - contrail-registry
+          nodeSelector:
+            node-role.kubernetes.io/worker: ""
+        serviceConfiguration:
+          cassandraInstance: cassandra1
+          controlInstance: control1
+          contrailStatusImage: hub.juniper.net/contrail-nightly/contrail-status:2008.121-rhel
+          containers:
+            - name: init
+              image: python:3.8.2-alpine
+            - name: nodeinit
+              image: hub.juniper.net/contrail-nightly/contrail-node-init:2008.121-rhel
+            - name: vrouteragent
+              image: hub.juniper.net/contrail-nightly/contrail-vrouter-agent:2008.121-rhel
+            - name: vroutercni
+              image: hub.juniper.net/contrail-nightly/contrail-kubernetes-cni-init:2008.121-rhel
+            - name: vrouterkernelbuildinit
+              image: hub.juniper.net/contrail-nightly/contrail-vrouter-kernel-build-init:2008.121-rhel
+            - name: vrouterkernelinit
+              image: hub.juniper.net/contrail-nightly/contrail-vrouter-kernel-init:2008.121-rhel
+            - name: multusconfig
+              image: busybox:1.31
```

Comment:
Same in this file, change registry to `hub.juniper.net/contrail`

Reply:
Done